README.md

```diff
@@ -73,17 +73,33 @@ for the docker worker in particular these are important:
 
 ## Running tests
 
-The `./test/test.sh` script is used to run individual tests which are
-suffixed with `_test.js` for example: `./test/test.sh test/integration/live_log_test.js`.
+There are a few components that must be configured for the tests to work properly (e.g. docker, kernel
```

Comment:
I just realized these steps are needed if you're on Mac OS X. If you're running Linux, the odds you can't run docker yourself are small.